safespace/src/components/Mood.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```

Comment:
this would be a perfect candidate for a functional component, no lifecycle methods or state is used, no reason to have it a class component